apps/designer/app/routes/designer.tsx

```diff
@@ -14,4 +15,12 @@ export const meta: MetaFunction = () => {
   return { title: "Webstudio" };
 };
 
+export const shouldRevalidate: ShouldRevalidateFunction = ({
```

Comment:
I think its not trivial to know why this is needed, please explain in the comment